rfcs/tag-component.md

```diff
@@ -0,0 +1,61 @@
+# Tag Component
+
+### Exports
+
+- `Tag`
+- `StyledRoot`
+- `StyledCross`
```

Comment:
Thoughts on calling this `StyledAction`? I like the idea of calling it "Action" because in theory someone could override the action to be whatever they want – for example a pencil icon if they want an "Edit" modal to popup.

Reply:
ok